packages/web/src/components/nav/desktop/PlaylistLibrary/PlaylistLibrary.tsx

```diff
@@ -55,49 +44,74 @@ export const PlaylistLibrary = (props: PlaylistLibraryProps) => {
   return (
     <ClassNames>
       {({ css }) => (
-        <Droppable
-          className={css({
+        <Flex
+          direction='column'
+          css={{
             position: 'relative',
-            // Drop Background
-            '::before': {
-              content: '""',
-              position: 'absolute',
-              top: -spacing.s,
-              bottom: -spacing.s,
-              left: 0,
-              right: 0,
-              backgroundColor: color.background.accent,
-              transition: `opacity ${motion.quick}`,
-              opacity: 0
-            },
-            '&.droppableLinkHover::before': {
-              opacity: 0.15
-            }
-          })}
-          hoverClassName='droppableLinkHover'
-          onDrop={handleDrop}
-          acceptedKinds={acceptedKinds}
+            maxHeight: '100%',
```

Comment:
i think using the attribute `h='100%' is preferred. limiting css when possible. same goes for w